cogs/csgo.py

```diff
@@ -389,13 +425,16 @@ async def pug(self, ctx: commands.Context, *args):
 
         if not self.pug.enabled:
             self.queue_check.start()
+            self.logger.info('Queue Starting Back')
 
     @pug.error
     async def pug_error(self, ctx: commands.Context, error: Exception):
         if isinstance(error, commands.CommandError):
             await ctx.send(str(error))
-        traceback.print_exc()
+            self.logger.warning(str(error))
+        self.logger.exception(f'{ctx.command} caused an exception')
 
+    @staticmethod
     def player_veto_embed(self, message_text, players_text, team1, team1_captain, team2, team2_captain):
```

Comment:
```suggestion
    @staticmethod
    def player_veto_embed(message_text, players_text, team1, team1_captain, team2, team2_captain):
```
No longer need reference to self since it is now a static method

Reply:
Ignore